test/io/corrado/qio_buf_optimization_perf/large_binary_io.chpl

```diff
@@ -0,0 +1,67 @@
+use IO;
+import Time.stopwatch, FileSystem.remove;
+
+config const n : uint(64) = 2048*8,
+             nn = 10000;
```

Comment:
Probably not the biggest deal, but these having similar names confused me a bit.